src/containers/execution-environment-list/execution_environment_list.tsx

```
@@ -467,15 +467,8 @@ class ExecutionEnvironmentList extends React.Component<
                 () => this.queryEnvironments(),
               );
             })
-            .catch((err) => {
-              this.setState({
-                formError: err.response.data.errors.map((error) => {
-                  return {
-                    title: error.title,
-                    detail: error.source.parameter + ': ' + error.detail,
-                  };
-                }),
-              });
+            .catch(() => {
```

we're not returning the promise anywhere, so an empty catch is not needed?

---

I now catched the second promise that contains 'return Promise.all([...'. So this outside catch is now not needed.